### ISSUE TITLE
Rockfield theme - Proceed to checkout button doesn't have any styles, shows up as a link

### ISSUE THREAD

**Issue by vinnykaur** [Automattic/themes#8467]
### Quick summary

When Rockfield theme is active, `Proceed to Checkout` shows up as a link. It doesn't have any button styles like the rest of the buttons on the site. 

### Steps to reproduce

1. Activate Rockfield theme on an eCommerce site.
2. Add a product and view cart page.
3. Notice the Proceed to checkout button. It appears as a link rather than a button.

![Image](https://github.com/user-attachments/assets/d2538d5e-75d5-414f-ab70-54b72b9d779a)

### A clear and concise description of what you expected to happen.

The Proceed to Checkout should be rendered as a button similar to the rest of the buttons to the site. 

### What actually happened

The Proceed to Checkout displayed as a link to a text. 

### Impact

One

### Available workarounds?

No but the platform is still usable

### If the above answer is "Yes...", outline the workaround.

I shared the below CSS with the user to make the link look like a button:

```
/* Customize cart link to add a background and hover zd-9066244-ck*/
.wc-block-cart__submit-button {
    color: #fff ;
    background: #8b4513;
}
.wc-block-cart__submit-button.contained:hover {
    opacity: .9;
    color: #fff;
}
```

### Platform (Simple and/or Atomic)

Atomic

**Comment by vinnykaur** [comment]
User reported here: 9066244-zd-a8c

**Comment by Robertght** [comment]
I'm going to mark this as Triaged on the One Board, as I can see we have someone assigned to it.

@ouikhuan feel free to do the same in the future with new issues. Thanks!

**Comment by ouikhuan** [comment]
Fixed in #8469
